### DISCUSSION TITLE
TUNING IN VFO MODE QUANSHENG uv-k5(8)

### DISCUSSION THREAD

**Discussion by mark1161** [egzumer/uv-k5-firmware-custom#363] [Q&A]
I like this firmware update.  Admittedly, I am a noob with this radio but have been a ham radio operator for a long time.  I am trying to figure out, when in HF, how to precisely tune the band.  When I press either B or C (up/down) buttons I am going up or down by at least 2 mhZ.  I have tried using the Step setting but it doesn't seem to work.  I must be missing something simple.   I have looked in the manual and discussions but can't find the answer detailed.  Any suggestions from the group?

73
Mark
KK2L

**Comment by AndrewTXJ** [comment]
change the step to a lower setting,

**Comment by ErikS-web** [comment]
Read the wiki about how to use the radio.

See [menu-control](https://github.com/egzumer/uv-k5-firmware-custom/wiki/Menu#menu-control)

Especially **_the last 2 lines_**

**Comment by mark1161** [comment]
After, I posted my question, I figured it out.  No problem now.  I am able to control the VFO on the radio.  Happy New Year.

 

73

 

Mark

KK2L
